prow/plank/controller.go

```diff
@@ -165,7 +165,7 @@ func (c *Controller) setPreviousReportState(pj prowapi.ProwJob) error {
 		latestPJ.Status.PrevReportStates = map[string]prowapi.ProwJobState{}
 	}
 	latestPJ.Status.PrevReportStates[reporter.GitHubReporterName] = latestPJ.Status.State
-	_, err = c.prowJobClient.Update(latestPJ)
+	_, err = c.kc.ReplaceProwJob(latestPJ.ObjectMeta.Name, latestPJ)
```

Comment:
what is this about?

Reply:
Sorry about that... I think I accidentally pulled in a change during rebase. When I squash that will go away.